script/download_rust_deps.py

```diff
@@ -144,10 +144,13 @@ def cargo_install(tool):
     cargo_args = []
     cargo_args.append(cargo_bin)
     cargo_args.append("install")
-    cargo_args.append(tool[0])
-    if len(tool) > 1:
+    cargo_args.append(tool["name"])
+    if "version" in tool:
         cargo_args.append("--version")
-        cargo_args.append(tool[1])
+        cargo_args.append(tool["version"])
+    if "features" in tool:
+        cargo_args.append("--features")
+        cargo_args.append(tool["features"])
```

Comment:
Just want to note that these could potentially be comma-separated features in the future if more than one is needed

Reply:
Indeed and it should work fine with the above since they're comma-separated and not whitespace-separated.